files/KoreBuild/modules/vstest/module.targets

```diff
@@ -98,7 +98,7 @@ Runs the VSTest on all projects in the ProjectsToTest itemgroup.
 
     <Message Text="%0AStarting test group: $(TestGroupName)/$(TargetFramework)" Importance="High" />
 
-    <RunDotNet Arguments="@(VSTestArgs)" IgnoreExitCode="true">
```

Comment:
We still need IgnoreExitCode=true set. Otherwise, the task will exit before we can report the TRX file location to TeamCity.